services/app-web/src/components/Header/UserLoginInfo/UserLoginInfo.tsx

```diff
@@ -17,15 +18,18 @@ const LoggedInUserInfo = (
     user: User,
     logout: LogoutHandlerT
 ): React.ReactElement => {
+    const stringConstants = useString()
+    const MAIL_TO_SUPPORT = stringConstants.MAIL_TO_SUPPORT
     return (
         <div className={styles.userInfo}>
+            <span>Contact </span>
             <a
-                href="mailto: mc-review@cms.hhs.gov, mc-review-team@truss.works"
+                href={`mailto: ${MAIL_TO_SUPPORT}, mc-review-team@truss.works`}
```

Comment:
This mailto link and the one in the footer open up a new tab with these console errors. This might be my browser because I tried opening it in incognito mode, and it worked fine.

If you can confirm this doesn't happen to you, then we can ignore this.

![image](https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/74f9f3d0-09f0-477b-964c-c3d478009420)

Reply:
Works for me.  Probably an extension on your browser, like you said.